src/Generator/ModelGenerator.php

```diff
@@ -118,6 +118,7 @@ private function generateQuery(string $baseNamespace, array $classmap, string $b
                     $parameter->setDefaultValue([]);
                 } else {
                     $parameter->setNullable($arg['nullable']);
+                    $parameter->setDefaultValue(null);
```

Comment:
Only if nullable, it should be default null?

Reply:
Yesss. Solved